### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by kenerwin88**
I modified the script to grab the latest saltstack version by default, instead of 2015..8.3 (which is now an older version).  Also replaced Write-Host with Write-Output. -- Err it looks like I accidently grabbed all the commits, I'm just trying to update bootstrap-salt.ps1 :(

### CODE REVIEW

**Comment by vutny** [comment]
@kenerwin88 Thanks... But if you would like to update only `bootstrap-salt.ps1` in `stable` branch, it's better to `git cherry-pick` the commit you need.
Eventually the `develop` branch should be fully merged into `stable`...

**Comment by vutny** [comment]
@kenerwin88 Could you be so kind to reopen your PR against `develop` branch only with your changes, please?

**Comment by jfindlay** [comment]
@kenerwin88, thanks for your work on this.  I'm going to close the pull request so that you can create another against develop when you have the time.  You are welcome to ask me if you need help rebasing the commits.

**Comment by vutny** [comment]
I think someone from SaltStack team could just take changes done by @kenerwin88 in `bootstrap-salt.ps1` file and apply the patch on the develop branch. There are trivial things: URL fix and bump of the installer version (2015.8.8-2 at the moment).

**Comment by jfindlay** [comment]
@vutny, forgive my obtuseness, but I have looked over the list of commits a few times and I don't see the changes you are referring to.  Are you able to cherry-pick the commits onto develop?

File: bootstrap-salt.ps1 [hunk 1]
```diff
@@ -40,14 +40,14 @@ is dynamically determined by the script.
...
-SaltStack Download Location - http://docs.saltstack.com/downloads/
+SaltStack Download Location - https://repo.saltstack.com/windows/
 #>
 [CmdletBinding()]
 Param(
   [Parameter(Mandatory=$false,ValueFromPipeline=$true)]
   # Doesn't support versions prior to "YYYY.M.R-B" 
   [ValidatePattern('^(201[0-9]\.[0-9]\.[0-9](\-\d{1})?)$')]
-  [string]$version = "2015.5.2",
+  [string]$version = "2015.8.3",
```
Comment (vutny) [inline bootstrap-salt.ps1:50]: @jfindlay I apologize, I mistakenly thought that here we got the changes made by @kenerwin88.
Now I clearly see there are no commits authored by him. Sorry for the inconvenience.

I guess all we can do here is just to bump Salt version...

Comment (jfindlay) [inline bootstrap-salt.ps1:50]: @vutny, no problem.  Thanks for working on this.

**Pull request closed without merging**